src/components/PostSection/index.jsx

```diff
@@ -86,8 +86,18 @@ const PostSection = ({
   const handleNotification = id => {
     PICKLY.createSingleNotification(id);
   };
-
+  const reportPost = () => {
+    console.log('reported');
+  };
```

Comment:
talk to the backend guys and make sure they finished this feature then add it and it will me ready for review.
it should finish at all